### PR TITLE
Modified tolerance to be able to customize.

### PR DESCRIPTION
**Pull request by hikouki**
🍮

I'd like to customize `tolerance` :)

### CODE REVIEW

File: scss/util/_color.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@
 /// @param {Color} $color2 - Second color to compare.
 ///
 /// @returns {Number} The contrast ratio of the compared colors.
-@function color-pick-contrast($base, $colors: ($white, $black), $tolerance: 0) {
+@function color-pick-contrast($base, $colors: ($white, $black), $tolerance: $global-color-pick-contrast-tolerance) {
```
Comment (IamManchanda) [inline scss/util/_color.scss]: Can you explain a bit about this advancement ? @hikouki

Comment (hikouki) [inline scss/util/_color.scss]: @IamManchanda 
Thank you for your review 🍣 
I want to set default `tolerance` .

`color-pick-contrast` function is used by following components:
- button
- badge
- callout
- label
- tabs

When I use this function, I thought if `tolerance` can be used more flexible,
it'll be more convenient. For example, even Button's background is a bit light color,
I want use light color to font too.

**Review by IamManchanda — APPROVED** [review]
Makes sense to me
@kball for decision on this!

File: scss/util/_color.scss (hunk 1 above)
Comment (IamManchanda) [inline scss/util/_color.scss]: Great. 
BTW.... Next time please tell this type of explanation in the  :arrow_double_up: above description! 
{{ Please dont leave description blank. It should tell what the code is doing. }}

**Comment by kball** [comment]
@andycochran I believe you wrote this originally, any objections?

**Comment by hikouki** [comment]
@kball @andycochran 
I'd like to know what the situation of the matter this. :bow:

**Comment by kball** [comment]
Merged! Will be in 6.4

**Comment by andycochran** [comment]
I went to address this and… 
Great work, @hikouki!

**Comment by andycochran** [comment]
We should probably note in the documentation how this functions. 

For example, the docs for [Label](http://foundation.zurb.com/sites/docs/label.html#text-colors) says: 
> The default settings meet WCAG 2.0 level AA contrast requirements. Be sure to check the contrast when changing color variables. To give all labels the same color text, set `$label-color` and `$label-color-alt` to the same value — but know that doing so may decrease accessibility.

That last sentence should maybe be changed to, "It's possible to adjust the `$global-color-pick-contrast-tolerance` variable so that labels that do not have enough contrast use `$label-color` instead of `$label-color-alt` — but know that doing so may decrease accessibility."

^ This will need to be noted for Badges, Buttons, and Labels. 

We also might indicate somewhere how to adjust the  `$global-color-pick-contrast-tolerance` variable. Should I change it to `0.5` or `42` or a billion? And how will that affect accessibility?

**Pull request merged**